Makefile

```diff
@@ -49,7 +49,9 @@ install: Makefile.inc hwdata.pc
 		install -m 644 $$foo $(DESTDIR)$(datadir)/$(NAME) ;\
 	done
 	mkdir -p -m 755 $(DESTDIR)$(libdir)/modprobe.d
-	install -m 644 -T blacklist.conf $(DESTDIR)$(libdir)/modprobe.d/dist-blacklist.conf
+	@if [ "$(blacklist)" == true ]; then \
+		install -m 644 -T blacklist.conf $(DESTDIR)$(libdir)/modprobe.d/dist-blacklist.conf ;\
+	fi; \
```

Comment:
Nit: I believe that final `\\` isn't necessary, as it would cause the following `mkdir` command to be interpreted as the same step as the blacklist stuff (ie, `make` won't print the `mkdir` command anymore).

Reply:
Your're right, I'll remove it, thanks!

Reply:
And I also moved creation of modprobe.d directory into the condition, I missed that initially.